scripts/build.js

```diff
@@ -4,8 +4,13 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const tasks = require('./tasks')
+const path = require('path')
 
-const buildDir = process.env.TARGET_GEN_DIR
+let buildDir = process.env.TARGET_GEN_DIR
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
I think we should really avoid the use of `NODE_ENV` anywhere anymore. It's often problematic because people set it in the local env. I think instead we should just set `process.env.TARGET_GEN_DIR` from within dev.js